test/unit/remote/cozy.js

```diff
@@ -56,13 +56,13 @@ describe('RemoteCozy', function () {
           let oldChanges = await remoteCozy.changes()
           let seq = oldChanges.last_seq
 
-          let dir = await builders.dir(remoteCozy).build()
-          let file = await builders.file(remoteCozy).inDir(dir).build()
+          let dir = await builders.dir().build()
+          let file = await builders.file().inDir(dir).build()
 
           let newChanges = await remoteCozy.changes(seq)
           let ids = newChanges.results.map(result => result.id).sort()
 
-          ids.should.eql([file._id, dir._id].sort())
+          ids.sort().should.eql([file._id, dir._id].sort())
```

Comment:
On L63, you set `ids` to `something.sort()`.
Then you `sort()` it again for the equality check.

Is it a mislead?